launch_ros/launch_ros/actions/node.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the Node action."""
+
+import logging
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import Text
+from typing import Tuple
```

Comment:
This line seems to result in a flake8 warning in the dev job: http://build.ros2.org/job/Ddev__launch_ros__ubuntu_bionic_amd64/21/consoleFull#console-section-17

@wjwwood Can you please double check this.

Reply:
This pr is ancient, if it didn't flake then, then why do I need to check this?

Reply:
The test fails in current `dev` builds but doesn't on ci.ros2.org. With double checki was referring to try to make it pass for both.

Reply:
That sounds like a CI issue, if they don't agree, but either way, wouldn't make more sense to open a new issue against this repository saying "this doesn't pass linters in the dev job", and then we can triage it? I don't think going back to the last pr to touch that file makes sense, especially if it's this old...

Reply:
Also, this code isn't in this repository any longer?

Reply:
(I was trying to look into it)

Reply:
The `dev` job uses the Debian package of `flake8` where as ci.ros.org uses the latest release from PyPI.

Reply:
It's probably because what ever version of the linter that's on the dev job doesn't understand the Python 3.5 version of type annotations:

https://github.com/ros2/launch_ros/blob/2d5d3eac831788d54c41069378733837b48b44c8/launch_ros/launch_ros/actions/node.py#L175

Reply:
Resolved by ros2/launch_ros#32.